### DISCUSSION TITLE
IQTREE stopped when Analyzing Big Data

### DISCUSSION THREAD

**Discussion by liangzhenhou** [iqtree/iqtree2#294] [Q&A]
Hi, thanks for the useful method! 
When I applied IQTREE-2 with >=5000 sequences and 600 sites, it stopped at "Generating 98 parsimony trees..." step. 
Could help me to fix it? I used the command "iqtree2 -s  sequence.phy -o 'root' "
Thank you!

**Comment by roblanf** [comment]
Hi @tenlives,

Can you define what you mean by 'stopped'? Can you provide the output files, and the stdout output?

My best guess is that it's just taking a long time, simply because generating 98 parsimony trees of >5000 sequences is a slow process. You might want to look into use the `-fast` option for datasets like this, or moving to software engineered more specifically for very large datasets like RaxML-NG, CMAPLE, or others.

Cheers,

Rob
